ansible/roles/ocp4-workload-dil-serverless/tasks/workload.yml

```diff
@@ -75,6 +81,79 @@
 
 - name: Provision {{ num_users }} User Namespaces
   include_tasks: provision_userprojects.yaml
+  vars:
+    smartcity_edge_project: "{{ ocp4_workload_dil_serverless_smartcity_edge_project }}"
+    smartcity_central_project: "{{ ocp4_workload_dil_serverless_smartcity_central_project }}"
+
+- name: Provision SmartCity Edge User Workloads
+  include_tasks: provision_smartcity_edge_loop.yaml
+  vars:
```

Comment:
Do we expect to have these changing frequently in the future?

Reply:
Not sure I understand.

Reply:
How frequently do we expect to change the values in the configuration files that require to be externalized as variables?